mmv1/products/alloydb/api.yaml

```diff
@@ -0,0 +1,265 @@
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: "error"
+        message: "message"
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: "clusterId"
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          The ID of the alloydb cluster.
+      - !ruby/object:Api::Type::String
+        name: "location"
+        input: true
+        url_param_only: true
+        description: |
+          The location where the alloydb cluster should reside.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: "name"
+        output: true
+        description: |
+          The name of the cluster resource.
+      - !ruby/object:Api::Type::String
+        name: "uid"
+        output: true
+        description: |
+          The system-generated UID of the resource.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "labels"
+        description: "User-defined labels for the alloydb cluster."
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "annotations"
+        description: "Annotations to allow client tools to store small amount of arbitrary data. This is distinct from labels."
+      - !ruby/object:Api::Type::String
+        name: "network"
+        required: true
+        description: |
+          The relative resource name of the VPC network on which the instance can be accessed. It is specified in the following form:
+
+          "projects/{projectNumber}/global/networks/{network_id}".
+      - !ruby/object:Api::Type::String
+        name: "displayName"
+        description: |
+          User-settable and human-readable display name for the Cluster.
+      - !ruby/object:Api::Type::String
+        name: "databaseVersion"
+        output: true
+        description: |
+          The database engine major version. This is an output-only field and it's populated at the Cluster creation time. This field cannot be changed after cluster creation.
+      - !ruby/object:Api::Type::NestedObject
+        name: "initialUser"
+        input: true
+        description: |
+          Initial user to setup during cluster creation.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "user"
+            description: |
+              The database username.
+          - !ruby/object:Api::Type::String
+            name: "password"
+            description: |
+              The initial password for the user.
+            required: true
+      - !ruby/object:Api::Type::NestedObject
+        name: "sslConfig"
+        description: |
+          SSL configuration for this AlloyDB Cluster.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: "sslMode"
+            description: |
+              The SSL mode. Specifies client-server SSL/TLS connection behavior.
+            default_value: :SSL_MODE_ALLOW
+            values:
+              - :SSL_MODE_UNSPECIFIED
```

Comment:
Generally we leave out the "unspecified" values in enums because they don't actually have a meaning on the API (and frequently trigger errors if you actually try to pass them). Would these be okay to remove?

Reply:
cool, will remove i just saw in other places we do have these

Reply:
done